client/src/lib.rs

```diff
@@ -17,8 +18,15 @@ pub mod rpc_deprecated_config;
 pub mod rpc_filter;
 pub mod rpc_request;
 pub mod rpc_response;
-pub mod rpc_sender;
+pub(crate) mod rpc_sender;
 pub mod spinner;
 pub mod thin_client;
 pub mod tpu_client;
 pub mod transaction_executor;
+
+pub mod mock_sender_for_cli {
```

Comment:
This is yucky, but I didn't want to export the senders (mock_sender, http_sender, rpc_sender) from the crate anymore as they're implementation details, and this was the only thing required by the rest of the monorepo.